src/test/java/com/faforever/client/game/GamesTableControllerTest.java

```diff
@@ -41,4 +54,36 @@ public void test() throws Exception {
         GameBuilder.create().defaultValues().state(GameStatus.CLOSED).get()
     ));
   }
+
+  @Test
+  public void testKeepsSorting() {
+    preferencesService.getPreferences().setGameListSorting(FXCollections.observableArrayList(
+        new Pair<>("hostColumn", SortType.DESCENDING)
+    ));
+
+    instance.initializeGameTable(FXCollections.observableArrayList(
+        GameBuilder.create().defaultValues().get(),
+        GameBuilder.create().defaultValues().state(GameStatus.CLOSED).get()
+    ));
+
+    assertThat(instance.gamesTable.getSortOrder(), hasSize(1));
+    assertThat(instance.gamesTable.getSortOrder().get(0).getId(), is("hostColumn"));
+    assertThat(instance.gamesTable.getSortOrder().get(0).getSortType(), is(SortType.DESCENDING));
+  }
+
+  @Test
+  public void testSortingUpdatesPreferences() {
+    assertThat(preferencesService.getPreferences().getGameListSorting(), hasSize(0));
+
+    instance.initializeGameTable(FXCollections.observableArrayList(
+        GameBuilder.create().defaultValues().get(),
+        GameBuilder.create().defaultValues().state(GameStatus.CLOSED).get()
+    ));
+
+    TableColumn<Game, ?> column = instance.gamesTable.getColumns().get(0);
+    column.setSortType(SortType.ASCENDING);
+    instance.gamesTable.getSortOrder().add(column);
+
+    assertThat(preferencesService.getPreferences().getGameListSorting(), hasSize(1));
```

Comment:
@themasch did u join our slack? What's your name there? If not just read the wiki and join via the link ;)